framework/source/class/qx/ui/table/model/Filtered.js

```diff
@@ -99,6 +99,7 @@ qx.Class.define("qx.ui.table.model.Filtered",
      * @param the_needle {String} String to search
      * @param the_haystack {Array} Array, which should be searched
      * @return {Boolean} whether the search string was found.
+     * @deprecated {6.0} Use qx.lang.Array.contains instead
```

Comment:
I just realized that I can replace the code by a call to qx.lang.Array.contains.
But does it make sense to keep this function alive ?